tuning/data/data_preprocessing_utils.py

```diff
@@ -49,12 +55,27 @@ def get_data_collator(
             str representing the human response in a chat template
         is_padding_free: bool
             if padding free plugin is used or not
+        text_field_name: str
```

Comment:
can we call this `dataset_text_field` if the name is not related to some terminology of vision language models. 
We have been using `dataset_text_field` inside our code at many places. 

similarly maybe update the name for `image_field_name`

Reply:
Would having `dataset_text_field` used for different purposes depending on the model type cause confusion? I think as long as we document it correctly then it should be fine since for text datasets using `dataset_text_field` is for the field with single formatted sequence whereas for text+image dataset it will be for the text field vs image field that seems good